src/Illuminate/Broadcasting/Broadcasters/PusherBroadcaster.php

```diff
@@ -119,9 +119,9 @@ public function broadcast(array $channels, $event, array $payload = [])
         }
 
         throw new BroadcastException(
-            empty($response['body'])
-                ? 'Failed to connect to Pusher.'
-                : sprintf('Pusher error: %s.', $response['status'])
+            isset($response['body'])
```

Comment:
I think isset() is not going to work here. The variable $response['body'] is a result of curl_exec(), always returning a boolean value. So isset() will always return true.

Reply:
We fixed this. Thanks!